framework/src/transfers/MultiAppGeneralFieldTransfer.C

```diff
@@ -688,7 +696,18 @@ MultiAppGeneralFieldTransfer::cacheIncomingInterpVals(
                            incoming_vals[val_offset].second))
         {
           // Keep track of distance and value
-          const auto p = point_requests[val_offset] - _to_positions[problem_id];
+          const auto & to_transform = _to_transforms[getGlobalTargetAppIndex(problem_id)];
+          Point p;
+          if (to_transform->hasNonTranslationTransformation())
+          {
+            if (!_skip_coordinate_collapsing)
+              mooseInfo("Search value conflict cannot find the origin point due to the "
+                        "non-uniqueness of the coordinate collapsing reverse mapping");
+            p = point_requests[val_offset] - _to_positions[problem_id];
```

Comment:
I don't understand this. We don't use the transform if there is non-translation information? But we do if it doesn't?

Reply:
changed. it s really the coordinate collapse back I dont want to do, because it errors on RZ -> XYZ but I do want to be doing that direction for now even if it s ill posed. maybe I shouldnt..

Reply:
it s for the search value conflict detection, it s not for the main values